build.sbt

```diff
@@ -1,68 +1,25 @@
-lazy val root = (project in file("."))
-  .settings(mainClass in Universal := Some("org.scalaexercises.evaluator.EvaluatorServer"))
-  .settings(stage <<= (stage in Universal in `evaluator-server`))
-  .settings(noPublishSettings: _*)
-  .aggregate(
-    `evaluator-server`,
-    `evaluator-shared-jvm`,
-    `evaluator-shared-js`,
-    `evaluator-client-jvm`,
-    `evaluator-client-js`)
-
-lazy val `evaluator-shared` = (crossProject in file("shared"))
+lazy val `evaluator-shared` = (project in file("shared"))
   .enablePlugins(AutomateHeaderPlugin)
   .settings(name := "evaluator-shared")
 
-lazy val `evaluator-shared-jvm` = `evaluator-shared`.jvm
-lazy val `evaluator-shared-js`  = `evaluator-shared`.js
-
-lazy val `evaluator-client` = (crossProject in file("client"))
+lazy val `evaluator-client` = (project in file("client"))
```

Comment:
I guess we can remove this module since it was moved to the scala-exercises server